data_tools/buildhub/jobs/scrape_archives.py

```diff
@@ -29,7 +30,8 @@
 def publish_records(client, records):
     with client.batch() as batch:
         for record in records:
-            batch.create_record(record)
+            record_id = build_record_id(record['download']['url'])
```

Comment:
I would do this in the `archive()` function instead since the `id` field can be put in the `data` mapping.